lib/fedora_migrate/repository_migrator.rb

```diff
@@ -67,9 +71,8 @@ def qualifying_object object
       return object if name.match(namespace)
     end
 
-    def find_or_create_report source
```

Comment:
I'm not sure why you change the name here. It seems like a more abstract name would be preferable, that way someone could come along and change the implementation later.